### DISCUSSION TITLE
Welcome to contributors to Dynamics 365 guidance content

### DISCUSSION THREAD

**Discussion by edupont04** [microsoft/dynamics365patternspractices#3] [Announcements]
In 2023, Microsoft starts publishing a new type of content for Microsoft Dynamics 365: Guidance content! This repo contains templates to help internal and external subject matter experts contribute to this content.

Learn more at [Dynamics 365 contributor guide](https://learn.microsoft.com/en-us/dynamics365/get-started/contribute#dynamics-365-guidance-content).

**Comment by mikaelstenroth** [comment]
Looking forward for more news on this!

**Reply by edupont04** [reply]
More news coming up soon - probably within a few days, I expect.

**Reply by edupont04** [reply]
Updated contributor guide with more information about the templates here: https://learn.microsoft.com/en-us/dynamics365/get-started/contribute
